airflow/hooks/hive_hooks.py

```diff
@@ -44,7 +44,8 @@ def get_context_from_env_var():
     """
     Extract context from env variable, e.g. dag_id, task_id and execution_date,
     so that they can be used inside BashOperator and PythonOperator.
-    :return: The context of interest.
+
+    :return The context of interest.
```

Comment:
@milton0825 , I'm having a further check and may you share more details about this change? Not getting it.

Reply:
I think we need an empty line between the description and the arg hint.

Reply:
But you removed the`:` after `return`? Any reason for it?

Reply:
I thought the right format is `:return` but looks like I was wrong after I did a search on Google.